packages/ui/src/modal-building-code-content/ModalBuildingCodeContent.tsx

```diff
@@ -80,6 +90,7 @@ const BuildingCodeContent: React.FC<BuildingCodeContentProps> = ({
                 ? "ui-ModalSide--SectionHighlighted"
                 : ""
             }`}
+            tabIndex={0}
```

Comment:
We might have to figure out a different solution here as these are now just tab-able at all times, but nothing is said. It would be confusing for the user I think if they press tab a few times while going through this content with nothing being read or happening verbally.

Reply:
So, for me the screen reader does read if I put the tabIndex on these. It does NOT read the definition if I tab to a new definition when I don't have `tabIndex{0}`
This video shows the current state for me:
https://github.com/user-attachments/assets/3fc1c2f9-44bb-4974-ba49-63babcfbdba6

This video shows how it behaves if I remove the tabIndex:
https://github.com/user-attachments/assets/2aecb6ca-268b-47aa-a1e6-e9f0fe1126d1